### DISCUSSION TITLE
I want to move all the repos under my Organisation to my user and then remove the Organisation

### DISCUSSION THREAD

**Discussion by VW-Talent** [community/community#39680] [Repositories]
### Select Topic Area

General

### Body

I want to remove my Organisation as we only need one account which I want our private repos to sit under, please.

Previously I've tried to transfer ownership of them from the organisation to the account but I keep getting this error:
![image](https://user-images.githubusercontent.com/106469554/203122428-8a486a20-0686-4607-83fa-48e30ef27bf1.png)

**Comment by cx0y** [comment]


**Reply by VW-Talent** [reply]
Hi @cx0y thanks for this but the main reason I'm asking here is because I've tried this and it gives me this error:
![image](https://user-images.githubusercontent.com/106469554/203122248-288320f4-9309-4706-8ce1-a4f97567885f.png)

I will include this in the body of the OP so people are aware. :)
